src/app/components/shell/cbar/community/community.ts

```diff
@@ -76,14 +77,23 @@ export default class AppShellCbarCommunity extends Vue {
 	}
 
 	get shouldShowLeave() {
-		return !this.community.hasPerms();
+		return !this.community.hasPerms() && !!this.community.is_member;
+	}
+
+	get shouldShowJoin() {
+		return !this.shouldShowLeave;
```

Comment:
We should also be checking whether or not the user is a mod/collaborator of a community here. It currently shows both `Join` and `Moderate` for the context menu of cbar items that I have collaborator status with.

Reply:
`Moderate` is for backend moderation.
For a community collaborator, "Edit Community" is shown instead of "Join Community".